MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.AppCompat/MvvmCross.Droid.Support.V7.AppCompat.csproj

```diff
@@ -138,6 +139,9 @@
       <Name>MvvmCross.Droid.Support.Core.UI</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <AndroidResource Include="Resources\Xml\Empty.xml" />
```

Comment:
How?  It was already generating the `Resource.Designer.cs` file but until I added that `Empty.xml` it was 0 kb.